<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"> <class>DeviceControlMainPage</class> <widget class="QWidget" name="DeviceControlMainPage">  <property name="geometry">   <rect>    <x>0</x>    <y>0</y>    <width>1500</width>    <height>690</height>   </rect>  </property>  <property name="minimumSize">   <size>    <width>1330</width>    <height>0</height>   </size>  </property>  <property name="windowTitle">   <string>Form</string>  </property>  <property name="styleSheet">   <string notr="true"/>  </property>  <layout class="QVBoxLayout" name="verticalLayout">   <item>    <widget class="QWidget" name="widget_head" native="true">     <property name="sizePolicy">      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">       <horstretch>0</horstretch>       <verstretch>0</verstretch>      </sizepolicy>     </property>     <property name="minimumSize">      <size>       <width>0</width>       <height>80</height>      </size>     </property>     <property name="maximumSize">      <size>       <width>16777215</width>       <height>80</height>      </size>     </property>     <layout class="QHBoxLayout" name="horizontalLayout_2">      <item>       <widget class="QPushButton" name="pushButton_start">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>start</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_stop">        <property name="enabled">         <bool>false</bool>        </property>        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>stop</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_reset">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>reset</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_1">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>Test</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_2">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>Test IPU</string>        </property>       </widget>      </item>      <item>       <spacer name="horizontalSpacer">        <property name="orientation">         <enum>Qt::Horizontal</enum>        </property>        <property name="sizeHint" stdset="0">         <size>          <width>40</width>          <height>20</height>         </size>        </property>       </spacer>      </item>     </layout>    </widget>   </item>   <item>    <widget class="QListWidget" name="listWidget">     <property name="sizePolicy">      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">       <horstretch>0</horstretch>       <verstretch>0</verstretch>      </sizepolicy>     </property>     <property name="minimumSize">      <size>       <width>1350</width>       <height>180</height>      </size>     </property>     <property name="maximumSize">      <size>       <width>16777215</width>       <height>180</height>      </size>     </property>     <property name="alternatingRowColors">      <bool>true</bool>     </property>    </widget>   </item>   <item>    <widget class="QWidget" name="widget" native="true">     <property name="sizePolicy">      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">       <horstretch>0</horstretch>       <verstretch>0</verstretch>      </sizepolicy>     </property>     <widget class="QGroupBox" name="groupBox_s1">      <property name="geometry">       <rect>        <x>1011</x>        <y>50</y>        <width>330</width>        <height>181</height>       </rect>      </property>      <property name="title">       <string>Station1</string>      </property>      <property name="alignment">       <set>Qt::AlignCenter</set>      </property>      <layout class="QHBoxLayout" name="horizontalLayout">       <property name="spacing">        <number>0</number>       </property>       <item>        <widget class="QWidget" name="widget_process_7" native="true">         <layout class="QVBoxLayout" name="verticalLayout_9">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>50</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_9">            <property name="text">             <string>IN</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s1_in">            <property name="styleSheet">             <string notr="true">color: rgb(255, 0, 0);</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_2" native="true">         <layout class="QVBoxLayout" name="verticalLayout_3">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>4</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_4">            <property name="text">             <string>Mixing</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLCDNumber" name="lcdNumber_s1_mixing_remaining">            <property name="value" stdset="0">             <double>0.000000000000000</double>            </property>            <property name="intValue" stdset="0">             <number>0</number>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s1_mixing">            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_1" native="true">         <layout class="QVBoxLayout" name="verticalLayout_2">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>4</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_3">            <property name="text">             <string>Sampling</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLCDNumber" name="lcdNumber_s1_sampling_remaining">            <property name="value" stdset="0">             <double>0.000000000000000</double>            </property>            <property name="intValue" stdset="0">             <number>0</number>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s1_sampling">            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_6" native="true">         <layout class="QVBoxLayout" name="verticalLayout_8">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>50</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_8">            <property name="text">             <string>OUT</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s1_out">            <property name="styleSheet">             <string notr="true">color: rgb(0, 0, 255);</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>      </layout>     </widget>     <widget class="QGroupBox" name="groupBox_s3">      <property name="geometry">       <rect>        <x>110</x>        <y>50</y>        <width>201</width>        <height>181</height>       </rect>      </property>      <property name="title">       <string>Station3</string>      </property>      <property name="alignment">       <set>Qt::AlignCenter</set>      </property>      <layout class="QHBoxLayout" name="horizontalLayout_5">       <item>        <widget class="QWidget" name="widget_process_5" native="true">         <layout class="QVBoxLayout" name="verticalLayout_6">          <item>           <widget class="QLabel" name="label_7">            <property name="text">             <string>Smear</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLCDNumber" name="lcdNumber_smear_remaining">            <property name="digitCount">             <number>5</number>            </property>            <property name="intValue" stdset="0">             <number>0</number>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_smear">            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>      </layout>     </widget>     <widget class="QWidget" name="widget_2" native="true">      <property name="geometry">       <rect>        <x>90</x>        <y>270</y>        <width>1291</width>        <height>101</height>       </rect>      </property>      <layout class="QVBoxLayout" name="verticalLayout_7">       <item>        <widget class="QSlider" name="horizontalSlider_cart1">         <property name="maximum">          <number>100</number>         </property>         <property name="sliderPosition">          <number>0</number>         </property>         <property name="orientation">          <enum>Qt::Horizontal</enum>         </property>         <property name="invertedAppearance">          <bool>true</bool>         </property>         <property name="invertedControls">          <bool>false</bool>         </property>         <property name="tickPosition">          <enum>QSlider::TicksAbove</enum>         </property>        </widget>       </item>       <item>        <widget class="QSlider" name="horizontalSlider_cart2">         <property name="maximum">          <number>100</number>         </property>         <property name="sliderPosition">          <number>0</number>         </property>         <property name="orientation">          <enum>Qt::Horizontal</enum>         </property>         <property name="invertedAppearance">          <bool>true</bool>         </property>         <property name="invertedControls">          <bool>false</bool>         </property>         <property name="tickPosition">          <enum>QSlider::TicksAbove</enum>         </property>        </widget>       </item>      </layout>     </widget>     <widget class="QGroupBox" name="groupBox_s2">      <property name="geometry">       <rect>        <x>660</x>        <y>50</y>        <width>330</width>        <height>181</height>       </rect>      </property>      <property name="title">       <string>Station2</string>      </property>      <property name="alignment">       <set>Qt::AlignCenter</set>      </property>      <layout class="QHBoxLayout" name="horizontalLayout_6">       <property name="spacing">        <number>0</number>       </property>       <item>        <widget class="QWidget" name="widget_process_8" native="true">         <layout class="QVBoxLayout" name="verticalLayout_10">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>50</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_10">            <property name="text">             <string>IN</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s2_in">            <property name="styleSheet">             <string notr="true">color: rgb(255, 0, 0);</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_9" native="true">         <layout class="QVBoxLayout" name="verticalLayout_11">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>4</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_11">            <property name="text">             <string>Mixing</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLCDNumber" name="lcdNumber_s2_mixing_remaining">            <property name="value" stdset="0">             <double>0.000000000000000</double>            </property>            <property name="intValue" stdset="0">             <number>0</number>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s2_mixing">            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_10" native="true">         <layout class="QVBoxLayout" name="verticalLayout_12">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>4</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_12">            <property name="text">             <string>Sampling</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLCDNumber" name="lcdNumber_s2_sampling_remaining">            <property name="value" stdset="0">             <double>0.000000000000000</double>            </property>            <property name="intValue" stdset="0">             <number>0</number>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s2_sampling">            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QWidget" name="widget_process_11" native="true">         <layout class="QVBoxLayout" name="verticalLayout_13">          <property name="leftMargin">           <number>4</number>          </property>          <property name="topMargin">           <number>50</number>          </property>          <property name="rightMargin">           <number>4</number>          </property>          <property name="bottomMargin">           <number>4</number>          </property>          <item>           <widget class="QLabel" name="label_13">            <property name="text">             <string>OUT</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_s2_out">            <property name="styleSheet">             <string notr="true">color: rgb(0, 0, 255);</string>            </property>            <property name="alignment">             <set>Qt::AlignCenter</set>            </property>            <property name="readOnly">             <bool>true</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>      </layout>     </widget>    </widget>   </item>  </layout> </widget> <resources/> <connections/></ui>